sdk/translation/Azure.AI.Translation.Document/api/Azure.AI.Translation.Document.netstandard2.0.cs

```diff
@@ -71,7 +71,7 @@ public partial class DocumentTranslationInput
         public Azure.AI.Translation.Document.TranslationSource Source { get { throw null; } }
```

Comment:
I think just needs a changelog and maybe consider adding a test to make sure categoryId is set?

Reply:
Added